doc/index.rst

```diff
@@ -176,7 +176,7 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    code/qml_drawer
    code/qml_fourier
    code/qml_gradients
-   code/qml_grouping
+   code/qml_pauli
```

Comment:
Needs to be alphabetical.  This should be moved below `qml_ops_op_math`.

Reply:
Done 👍🏼